target/linux/octeon/base-files/lib/preinit/79_move_config

```diff
@@ -1,19 +1,23 @@
 # Copyright (C) 2014 OpenWrt.org
 
-move_config() {
-	. /lib/functions.sh
+octeon_move_config() {
 	. /lib/upgrade/common.sh
+	local device="$1"
+	[ -n "$device" ] && [ -b "$device" ] && {
```

Comment:
Do we need this additional check `-b`? It has worked well without it so far ...

Reply:
we expect it to be a block device and it should be a block device.
what if it is not? that could be a problem.
it worked before but mistakes could be made and it could fire right here when someone could try to mount a device that is not a block device.